### PR TITLE
🌟 Version 1.0.01 release

### PR DESCRIPTION
**Pull request by m-sullivan7**
Includes:

- Homepage update to hero H2

### CODE REVIEW

**Comment by m-sullivan7** [comment]
@jbum This should just be content changes, but we've also pushed since making the branch. Hopefully that doesn't create any conflicts.

I don't know for these releases who gets the honor of pushing the merge button once you've approved. 😉

**Review by jbum — APPROVED** [review]
Looks good.  Modified URL, additional reading, and minor text tweaks.

**Pull request merged**
